crates/storage/store/storage.rs

```diff
@@ -70,14 +68,13 @@ impl AccountUpdate {
 }
 
 impl Store {
-    pub fn new(path: &str, engine_type: EngineType) -> Result<Self, StoreError> {
+    pub fn new(_path: &str, engine_type: EngineType) -> Result<Self, StoreError> {
         info!("Starting storage engine ({engine_type:?})");
         let store = match engine_type {
             #[cfg(feature = "libmdbx")]
             EngineType::Libmdbx => Self {
-                engine: Arc::new(LibmdbxStore::new(path)?),
+                engine: Arc::new(LibmdbxStore::new(_path)?),
```

Comment:
why did you change to _ if you're using it

Reply:
because the `path` is only being used if the `libmdbx` feature is set and clippy was throwing lint warning with the flag disabled,  but since we have it as default i will change it back